File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/IdAlertsBody.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class IdAlertsBody {
    /// <summary>
    /// The ID of the alert to delete.
    /// </summary>
    /// <value>The ID of the alert to delete.</value>
    [DataMember(Name="alert_id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "alert_id")]
    public int? AlertId { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class IdAlertsBody {\n");
      sb.Append("  AlertId: ").Append(AlertId).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/id_alerts_body.dart
part of swagger.api;

class IdAlertsBody {
  /* The ID of the alert to delete. */
  int alertId = null;

  IdAlertsBody();

  @override
  String toString() {
    return 'IdAlertsBody[alertId=$alertId, ]';
  }

  IdAlertsBody.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    alertId = json['alert_id'];
  }

  Map<String, dynamic> toJson() {
    return {
      'alert_id': alertId
     };
  }

  static List<IdAlertsBody> listFromJson(List<dynamic> json) {
    return json == null ? new List<IdAlertsBody>() : json.map((value) => new IdAlertsBody.fromJson(value)).toList();
  }

  static Map<String, IdAlertsBody> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, IdAlertsBody>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new IdAlertsBody.fromJson(value));
    }
    return map;
  }
}
